deb/build_deb.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# Script for deb packaging dvbtee
+#
+# Tested on Ubuntu 14.04
+#
+# Author: Pasha Mesh <pasha.mesh@gmail.com>
+#
+
+PATH='/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games'
```

Comment:
are all of these directories necessary for this script to do its job?  specifically, '/usr/games' ?